requirements.txt

```diff
@@ -6,3 +6,4 @@ pyrsistent
 python-mimeparse
 unittest2>=1.0.0
 traceback2
+six
```

Comment:
What version of six introduced the functions you're using? I've seen bug reports galore with mock due to folk having old distro copies installed :(